accumulo/src/main/scala/geotrellis/spark/io/accumulo/AccumuloRDDWriter.scala

```diff
@@ -44,14 +55,41 @@ object AccumuloRDDWriter {
 
     instance.ensureTableExists(table)
 
+    val kwWriterSchema = KryoWrapper(writerSchema)
+
     val kvPairs: RDD[(Key, Value)] =
       raster
         // Call groupBy with numPartitions; if called without that argument or a partitioner,
         // groupBy will reuse the partitioner on the parent RDD if it is set, which could be typed
         // on a key type that may no longer by valid for the key type of the resulting RDD.
         .groupBy({ row => encodeKey(row._1) }, numPartitions = raster.partitions.length)
-        .map { case (key, pairs) =>
-          (key, new Value(AvroEncoder.toBinary(pairs.toVector)(codec)))
+        .map { case (key, _kvs1) =>
+          val kvs1: Vector[(K,V)] = _kvs1.toVector
+          val kvs2: Vector[(K,V)] =
+            if (mergeFunc != None) {
```

Comment:
Minor nitpick, so ignore if you don't care, but below you do pattern matching on `Option[(V,V) => V]` and here using identity boolean check. Consistency might help readability. I prefer pattern matching or `mergeFunc.nonEmpty` myself.